app/views/spotlight/exhibits/edit.html.erb

```diff
@@ -6,6 +6,9 @@
       <li role="presentation" class="active">
         <a href="#basic" aria-controls="basic" role="tab" data-toggle="tab"><%= t(:'.basic_settings.heading') %></a>
       </li>
+      <li role="presentation">
```

Comment:
Could we add + check an ability? `can :manage, current_exhibit.filters.build` maybe?